spec-views/after.rb

```diff
@@ -1,14 +0,0 @@
-require "yaml"
```

Comment:
Removed spec-views folder....no idea what there for or how to even run them....did we write our own custom integration framework or something?